config.yml

```diff
@@ -26,12 +26,29 @@ genesis:
         denom: uelys
     gov:
       deposit_params:
-        max_deposit_period: 20s
+        max_deposit_period: 172800s
         min_deposit:
           - amount: "10000000"
             denom: uelys
+      params:
+        burn_proposal_deposit_prevote: false
+        burn_vote_quorum: false
+        burn_vote_veto: true
+        max_deposit_period: 100s
+        min_deposit:
+          - amount: "10000000"
+            denom: uelys
+        min_initial_deposit_ratio: "0.000000000000000000"
+        quorum: "0.334000000000000000"
+        threshold: "0.500000000000000000"
+        veto_threshold: "0.334000000000000000"
+        voting_period: 100s
+      tally_params:
+        quorum: "0.334000000000000000"
+        threshold: "0.500000000000000000"
+        veto_threshold: "0.334000000000000000"
       voting_params:
-        voting_period: 20s
+        voting_period: 100s
```

Comment:
And voting period as well

Reply:
Yeah, I should had to update config.yml of gov modules. And it was 20s from Caner's configuration. Let us discuss about it.

Reply:
@kenta-elys same here thanks

Reply:
@cosmic-vagabond Done